src/Tribe/Views/V2/Views/Recent_Past_View.php

```diff
@@ -110,7 +140,10 @@ protected function setup_repository_args( Tribe__Context $context = null ) {
 		$this->repository = tribe_events();
 
 		$date                   = $context->get( 'event_date', 'now' );
-		$args['posts_per_page'] = $context->get( 'events_per_page', 3 );
+		//$args['posts_per_page'] = $context->get( 'events_per_page', 3 );
```

Comment:
* :warning: This comment is 71% valid code; is this commented out code?
* :no_entry_sign: No space found before comment text; expected "// $args['posts_per_page'] = $context->get( 'events_per_page', 3 );" but found "//$args['posts_per_page'] = $context->get( 'events_per_page', 3 );"

<sub>hash: db1ea7db7a8ea83ef60b2108656412b2</sub>